src/airhockey_vision/tests/test_trajectory.py

```diff
@@ -0,0 +1,42 @@
+from unittest import TestCase
+import pytest
+import numpy as np
+from airhockey_vision.puck_trajectory import TrajectoryCalculator
+
+class TestTrajectory(TestCase):
+    def test_first_run(self):
+        t = TrajectoryCalculator()
+        x,y = 30, 30
+        vector = t.calc_trajectory(x,y,1.0001)
+        assert vector[2] == vector[3] == 0
+        assert t.first_run == False
+        print(t.prev_time)
+
+    def test_future_pos(self):
+        x1,y1,x2,y2 = 30,30,31,35
+        time1, time2 = 1.005, 2.005
+        t = TrajectoryCalculator()
+        t.calc_trajectory(x1,y1,1.005)
+        x, y, xd, yd = t.calc_trajectory(x2,y2,2.005)
+        assert x == 31
+        assert y == 35
+        assert xd - 1 < 1e3
```

Comment:
you can use `pytest.approx` here instead, that would give you better error messages if things go wrong. see [test_puck_tracker:75](https://github.com/wsh32/airhockey/blob/main/src/airhockey_vision/tests/test_puck_tracker.py#L75) for an example